inference-engine/thirdparty/clDNN/src/graph_optimizer/reorder_inputs.cpp

```diff
@@ -432,10 +465,58 @@ void reorder_inputs::run(program_impl& p, layout_optimizer& lo, reorder_factory&
 
             fmt_map.at(node_ptr) = format::b_fs_yx_fsv16;
             fmt_map.at(conv_node.get_users().front()) = format::b_fs_yx_fsv16;
+
+            CLDNN_REORDER_INPUTS_PATTERN_MATCH_LOG("change int8 mvn->conv->mvn to b_fs_yx_fsv16", node_ptr->id());
```

Comment:
This WA is going to be removed in #1865. So do you think we'll still need `CLDNN_REORDER_INPUTS_PATTERN_MATCH_LOG` macro after that?

Reply:
I'm also planning to use it for coming fully connected switch to reorder_inputs, so I'd prefer to keep it in PR, if that's ok.
I will however remove it from this mvn WA to avoid conflicts.